File: Iniciante/1132 - Múltiplos de 13/Python/1132.py
def main():
    x = int(input())
    y = int(input())
    total = 0
    end = max(x, y)
    start = min(x, y)

    for i in range(start, end + 1):
        if i % 13 != 0:
            total += i

    print(total)

if __name__ == "__main__":
    main()

File: Iniciante/1132 - Múltiplos de 13/JavaScript/1132.js
function main() {
    const input = require('fs').readFileSync(0, 'utf-8').trim().split(/\s+/);
    const x = parseInt(input[0]);
    const y = parseInt(input[1]);
    let sum = 0;
    const end = Math.max(x, y);
    const start = Math.min(x, y);

    for (let i = start; i <= end; i++) {
        if (i % 13 !== 0) {
            sum += i;
        }
    }

    console.log(sum);
}

main();